backend/mlarchive/bin/scan_all.py

```diff
@@ -176,15 +176,23 @@ def archived_at():
     print('mismatches: {} of ({})'.format(mismatch, count))
 
 
-def archived_at_report(fix=False):
+def archived_at_report(start='2013-01-01', fix=False, verbose=False):
     """Find messages whose Archived-At header does not match url.
     --fix will populate Redirect table for mismatched URLs
+    --start, the date to start scan from, defaults to 2013-01-01
+
+    Examples:
+    ./scan_all.py archived_at_report
+    ./scan_all.py --start=2023-01-01 archived_at_report
+    ./scan_all.py --start=2023-01-20 --fix --verbose archived_at_report
     """
     from mlarchive.archive.models import Redirect
     from urllib.parse import urlparse
     from email.parser import BytesParser
     parser = BytesParser()
-    for message in Message.objects.filter(date__year__gte=2013).order_by('-date'):
+    start_date = parse(start)
+    print("start: {}    fix: {}    verbose: {}".format(start, fix, verbose))
```

Comment:
did you want to guard this with verbose?

Reply:
I have removed verbose altogether